README.md

```diff
@@ -1,18 +1,41 @@
 # Azure Exporter for Scrapy
+scrapy_azure_exporter is a Python package which can be used in integration with scrapy for exporting feeds to Azure Storage.
+## Description:
+The scrapy_azure_exporter takes inspiration from the built-in feed exporters in scrapy. The built-in exporters doesn't have support for Azure yet.
+So this scrapy_azure_exporter can be used whenever when we need the feeds to be exported to Azure Blob storage.
 
-## Requirements:
-- Python 3.8
-- Pipenv
+## Getting started:
+### Installation:
```

Comment:
Headers should not end in a colon (I realize this comes from the original content).

```suggestion
### Installation
```